libs/media-messages/schemas/payloads/media/media-service-asyncapi.yml

```diff
@@ -86,38 +86,56 @@ channels:
 components:
   messages:
     check-finish-ingest-item-command:
+      tags:
+        - name: aggregate-type:ingest-item
       contentType: application/json
       payload:
         $ref: 'commands/check-finish-ingest-item-command.json'
     check-finish-ingest-document-command:
+      tags:
+        - name: aggregate-type:ingest
       contentType: application/json
       payload:
         $ref: 'commands/check-finish-ingest-document-command.json'
     delete-entity-command:
+      tags:
+        - name: aggregate-type:entity
       contentType: application/json
       payload:
         $ref: 'commands/delete-entity-command.json'
     publish-entity-command:
+      tags:
+        - name: aggregate-type:entity
       contentType: application/json
       payload:
         $ref: 'commands/publish-entity-command.json'
     start-ingest-command:
+      tags:
+        - name: aggregate-type:ingest
```

Comment:
I think it would be more accurate to change to `aggregate-type:ingest-document`
in both cases where this is used (to align with `ingest-item`)